shared/teams/team/nav-header/index.js

```diff
@@ -97,3 +83,36 @@ export const HeaderTitle = (props: HeaderTitleProps) => (
     </Kb.Box2>
   </Kb.Box2>
 )
+
+type SubHeaderProps = {
+  onAddSelf: ?() => void,
+}
+
+export const SubHeader = (props: SubHeaderProps) =>
+  props.onAddSelf ? (
+    <Kb.Box2 direction="horizontal" style={styles.banner} fullWidth={true}>
+      <Kb.Banner
+        color="blue"
+        inline={true}
+        text="You are not a member of this team."
+        actions={[{onClick: props.onAddSelf, title: 'Add yourself'}]}
+      />
+    </Kb.Box2>
+  ) : null
+
+const styles = Styles.styleSheetCreate({
+  banner: {
+    ...Styles.padding(Styles.globalMargins.xsmall, Styles.globalMargins.xsmall, 0),
```

Comment:
:sheep: maybe remove the unnecessary spreads

Reply:
It's necessary for flow unfortunately, it complains about all the styles that exist in `_StylesCrossPlatform` but aren't returned by `Styles.padding`